functionary/core/api/v1/views/task.py

```diff
@@ -52,7 +58,14 @@ def get_serializer_class(self):
         description=(
             "Execute a function. The function to be executed can be defined either "
             "by supplying function as a string uuid, or function_name and "
-            "package_name."
+            "package_name. "
+            "In order to submit Tasks that take in file parameters, you must submit "
+            "requests to the API endpoint through the command line, and prefix the "
+            "file parameters with `file.`. The file parameters should not be placed "
+            "in the parameters argument. For example, "
+            "`-F 'function=1e8a7097-33b6-4ec4-87fc-52b7079caa76' "
```

Comment:
This documentation is not intended as instructions on how to use a specific tool to call the API, but rather documentation of how the endpoint works and what input it takes.  Basically you just need to describe the nature the of the `file.<parameter_name>` convention here and not how someone might use curl to call it.